CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to mozilla.ai any-llm
+
+Thank you for your interest in contributing to this repository! This project supports the mozilla.ai goal of empowering choice and transparency when it comes to LLM usage and selection.
+
+We welcome all kinds of contributions, from improving customization, to extending capabilities, to fixing bugs. Whether you’re an experienced developer or just starting out, your support is highly appreciated.
+
+## **Guidelines for Contributions**
+
+**Install**
+
+We recommend to use [uv](https://docs.astral.sh/uv/getting-started/installation/):
+
+```
+uv venv
+source .venv/bin/activate
+uv sync --dev --extra all
+```
+
+**Lint**
+
+Ensure all the checks pass:
+
+```bash
+pre-commit run --all-files
```

Comment:
This is what I use 🤷 
```suggestion
uv run pre-commit run --all-files --verbose
```

Reply:
totally. I think this makes sure that we use the local install of pre-commit even if there is one installed globally? This way someone can run the pre-commit even if they didn't source the env yet.